src/Symfony/Component/AssetMapper/ImportMap/Resolver/JsDelivrEsmResolver.php

```diff
@@ -72,7 +72,7 @@ public function resolvePackages(array $packagesToRequire): array
         $cssEntrypointResponses = [];
         foreach ($requiredPackages as $i => [$options, $response, $packageName, $filePath]) {
             if (200 !== $response->getStatusCode()) {
-                $errors[] = [$options->packageName, $response];
+                $errors[] = [$packageName, $response];
```

Comment:
I think this should stay the same. It’s only used in the error. And there, we’d want to mention the entire “package name” that the user has in their importmap.

The “package name” that is a package+path vs the “package name” that is actually the package name is a lot the root of the confusion. This is something we’re discussing on another PR to clarify.